src/main/kotlin/build/buf/intellij/index/ModuleKeyIndex.kt

```diff
@@ -97,7 +88,7 @@ class BufModuleKeyIndex : ScalarIndexExtension<ModuleKey>() {
             }
             val commit = bufModuleItem["commit"] ?: return null
             val digest = bufModuleItem["digest"]?.let {
-                ModuleDigest.parse(it).getOrThrow()
+                ModuleDigest.parse(it).getOrNull()
```

Comment:
When loading the buf CLI project I saw warnings because there are some test `buf.yaml` files with unsupported `b1` digests.